airbyte-integrations/connectors/source-twilio/source_twilio/schemas/conversation_messages.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "sid": {
+      "type": ["null", "string"]
+    },
+    "account_sid": {
+      "type": ["null", "string"]
+    },
+    "conversation_sid": {
+      "type": ["null", "string"]
+    },
+    "body": {
+      "type": ["null", "string"]
+    },
+    "media": {
+      "type": ["null", "string"]
+    },
+    "author": {
+      "type": ["null", "string"]
+    },
+    "participant_sid": {
+      "type": ["null", "string"]
+    },
+    "attributes": {
+      "type": ["null", "string"]
+    },
+    "date_created": {
+      "type": ["null", "string"]
+    },
+    "date_updated": {
+      "type": ["null", "string"]
+    },
+    "index": {
+      "type": "integer"
```

Comment:
please change to ["null", "integer"] to avoid possible schema problems in future